zcash_client_backend/src/proto/proposal.rs

```diff
@@ -116,6 +116,10 @@ pub mod proposed_input {
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct TransactionBalance {
     /// A list of change output values.
+    ///
+    /// Each `ChangeValue` for the transparent value pool must be consumed by
+    /// a subsequent step. These represent ephemeral outputs that will each be
+    /// given a unique t-address.
```

Comment:
This will not be true once fully-transparent transactions are supported, so I don't think that it should be documented as such here.

Reply:
Ephemeral `ChangeValue`s are now represented explicitly, so this is resolved.